tdm-fe/src/app/components/layout/ServiceDescription.tsx

```diff
@@ -0,0 +1,55 @@
+import './scss/ServiceDescription.scss';
+import DownloadIcon from '@mui/icons-material/Download';
+import EmailIcon from '@mui/icons-material/Email';
+import TaskAltIcon from '@mui/icons-material/TaskAlt';
+import { Button, Container } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+
+const ServiceDescription = () => {
+    const navigate = useNavigate();
+
+    const handleStartClick = () => {
+        navigate('/');
+    };
+
+    return (
+        <div className="service-description">
+            <Container>
+                <h2>IA Factory c'est quoi ?</h2>
+                <p className="description-text">
+                    IA Factory est une interface de chargement de corpus et d'exécution d'outils de TDM vous permettant 
+                    d'exploiter vos propres données en simplement 3 étapes. Nous garantissons l'anonymat, les données 
```

Comment:
Pour rester cohérent, on pourrait enlever le nombre d'étapes (en dessous, dans la maquette on en a 4).
```suggestion
                    d'exploiter vos propres données en quelques étapes. Nous garantissons l'anonymat, les données 
```